app/DiscordWebhook.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use GuzzleHttp\Client;
+
+class DiscordWebhook
+{
+    public static function notifyArchub(string $content)
+    {
+        $client = new Client(['headers' => ['Content-Type' => 'application/json']]);
+        $response = $client->request('POST', config('services.discord.archub_webhook'), [
+            'json' => ['content' => $content],
+        ]);
+    }
+
+    public static function notifyStaff(string $content)
+    {
+        $client = new Client(['headers' => ['Content-Type' => 'application/json']]);
+        $response = $client->request('POST', config('services.discord.staff_webhook'), [
+            'json' => ['content' => $content],
+        ]);
```

Comment:
Could be nice to use embeds, though that might be better suited for future PRs.

Reply:
Once larvael can be updated there will be a better HTTP library so I'll look at doing more complicated requests then
alternativelyI might just route info to SvenBot and rework all the messages at that point